test/unit/dune

```diff
@@ -1,4 +1,7 @@
 (test
  (name test_unit)
+ (deps
+  ../passing/tests/partial.ml
+  ../passing/tests/format_invalid_files_with_locations.ml)
```

Comment:
There's a read to `../../lib/Fmt_ast.ml` in `Test_translation_unit` but I think you can remove the read call.

Reply:
This patch is minimal to please the CI of https://github.com/ocaml/opam-repository/pull/23407 so I will merge it as it is.